manzi-guev/Teamwork
# #168880626 User signing Up

### What does this PR do
> -It helps the user to be able to sign up
### Descriptions of the tasks to be completed
> -Helping the user to be able to sign up and create an account
### How should this be manually tested
> -Start your server
> -Go on postman and use the right methods to use
> -When a user tries to sign up again there should be a message telling that the user already exists
### Screenshots
![image](https://user-images.githubusercontent.com/53451069/66038140-8c25c100-e511-11e9-9e83-98ccc9b05be4.png)
![image](https://user-images.githubusercontent.com/53451069/66038168-a1025480-e511-11e9-9212-406101ad6409.png)
![image](https://user-images.githubusercontent.com/53451069/66039259-a57c3c80-e514-11e9-8078-b240d5792f48.png)




### What are the relevant pivotal tracker stories
[#168880626](https://www.pivotaltracker.com/story/show/168880626)
